core/src/main/java/io/aiven/klaw/config/SecurityConfigNoSSO.java

```diff
@@ -163,7 +163,7 @@ private void loadAllUsers(
       try {
         String secPwd = userInfo.getPwd();
         if (secPwd != null && secPwd.equals("")) {
```

Comment:
should this be if (secPwd == null || (secPwd != null && secPwd.equals("")))

Reply:
In UserTeamsService we have this code to set pwd to ""
userInfo.setUserPassword("");

however updated the code as you suggested.